src/sentry/static/sentry/app/components/version.tsx

```diff
@@ -50,6 +50,10 @@ type Props = {
    * Ellipsis on overflow
    */
   truncate?: boolean;
+  /**
+   * Use releases v2 (for example when linking to detail page)
+   */
+  v2?: boolean;
```

Comment:
If an organization has releases-v2 should all links go to the v2 pages? If so you could wrap the Version component with `withOrganization()` so you have access to the current organization and not need to update all the callsites.

Reply:
I will definitely do it this way - it will be a part of a separate PR. This is a temporary solution (demo purposes) - for now, you go to releasesV2 detail page only from releasesV2 listing.